src/megaapi_impl.cpp

```diff
@@ -21132,8 +21140,50 @@ void MegaApiImpl::sendPendingRequests()
                 e = API_EARGS;
                 break;
             }
+
+            long long q = request->getTotalBytes();
+            int bs = request->getAccess();
+            int us = request->getNumDetails();
 #ifdef DEBUG
-            client->senddevcommand(command, email);
+            if (strcmp(command, "aodq")
+                  && strcmp(command, "tq")
+                  && strcmp(command, "bs")
+                  && strcmp(command, "us"))
+            {
+                e = API_EARGS;
+                break;
+            }
+
+            if (!strcmp(command, "tq"))
+            {
+                if (q < 0)
+                {
+                    e = API_EARGS;
+                    break;
+                }
+            }
+            else if (!strcmp(command, "bs"))
+            {
+                if (bs < -1 || bs > 2)
+                {
+                    e = API_EARGS;
+                    break;
+                }
+                if (!isBusinessAccount())
```

Comment:
This check might fail if the user attempts to set the `bs` for a different user by passing its email address. I think we should simply remove it.